socorro/external/es/crashstorage.py

```diff
@@ -650,3 +650,9 @@ def _submit_crash_to_elasticsearch(
                     exc_info=True,
                 )
                 raise
+
+    def document_exists(self, date, crash_id):
+        index = self.get_index_for_date(date)
+        # TODO: Are there different `doc_type`s that we use? Is `_all` fine?
+        # https://elasticsearch-py.readthedocs.io/en/1.9.0/api.html#elasticsearch.Elasticsearch.exists
+        return self.client.connection().exists(index=index, doc_type="_all", id=crash_id)
```

Comment:
This kind of breaks the API structure we've got here where crashstorage should use the connection context to do the things in Elasticsearch.

Instead of this, we can do what we do in the VerifyCrash API and create an `ESCrashStorage`, feed that into a `SuperSearch` and then use existing things to figure out if the item is in ES or not. We can do that entirely in the `load_processed_crash_into_es.py` script such that when we no longer need that script, we don't have anything additional lying around.

```python
import sys

from socorro import settings
from socorro.external.es.supersearch import SuperSearch
from socorro.external.es.super_search_fields import FIELDS
from socorro.libclass import build_instance_from_settings

crash_id = sys.argv[1]

crashstorage = build_instance_from_settings(settings.ES_STORAGE)
supersearch = SuperSearch(crashstorage)

params = {
    "_columns": ["uuid"],
    "_results_number": 1,
    "uuid": crash_id,
    "dont_cache": True,
    "refresh_cache": True,
    "_fields": FIELDS,
}
results = supersearch.get(**params)
exists = results["total"] == 1 and results["hits"][0]["uuid"] == crash_id

print(f"exists?: {exists}")
```

Reply:
Thanks for the suggestion! I had some trouble importing `SuperSearch` when I tried that last week, but I couldn't find anything in my notes about it, so I'll give it another shot.